lib/FullScreenView/FullScreenView.js

```diff
@@ -64,8 +69,10 @@ export const FullScreenView = ({
 
 FullScreenView.propTypes = {
   id: PropTypes.string,
-  onCancel: PropTypes.func.isRequired,
```

Comment:
Please, could you clarify: 
1. If onCancel is not required any more. Will it work always without errors?
2. Are there any cases when onCancel should not be provided?

Reply:
In case when `renderHeader` passes to the component (see https://github.com/folio-org/stripes-components/tree/master/lib/Pane) it will render Custom header with all needed handlers, and in that case onCancel is not needed. Firstly I suggested Viktor to remove all this props and leave only renderHeader prop according to docs, but after discussion was decided to make these props not required and left default renderHeader in case when renderHeader not passes

Reply:
Yeah, in case of the renderHeader is passed the `onCancel` is not required. I think what we can try here is to add custom validation logic for the prop.

```
  customProp: function(props, propName, componentName) {
    if (!/matchme/.test(props[propName])) {
      return new Error(
        'Invalid prop `' + propName + '` supplied to' +
        ' `' + componentName + '`. Validation failed.'
      );
    }
  },
```

Example from react [docs](https://reactjs.org/docs/typechecking-with-proptypes.html). I am okay to leave optional as even if it is not passed the app would not be broken.

Reply:
Do we really need it here)?

Reply:
Yeah, I understand the idea. Okay, it works for now. I think, these forms are very similar so it will be nice to add more default stuff in the future.